feat(Progress List)!: Make collapsing and expanding steps optional (#2538)
Amsterdam/design-system: storybook/src/components/ProgressList/ProgressList.docs.mdx

Current and upcoming steps then start expanded.
Completed ones are initially collapsed.
Use the `defaultCollapsed` prop on a step to override the default behaviour.

> The `onToggle` callback emits whether the Step is expanded.

Reviewer: Can you rephrase this a bit? The word “whether” is a bit confusing because it does not suggest that the emitting happens every time the expanded state changes.

Maybe:

```suggestion
The `onToggle` callback emits the expanded state whenever it changes.
```
